rest-api/dao/participant_dao.py

```diff
@@ -274,10 +274,10 @@ def add_missing_hpo_from_site(self, session, participant_id, site_id):
     participant = self.get_for_update(session, participant_id)
     if participant is None:
       raise BadRequest('No participant %r for HPO ID udpate.' % participant_id)
-    if participant.hpoId == site.hpoId:
-      return
 
     participant.hpoId = site.hpoId
+    participant.organizationId = site.organizationId
```

Comment:
Please add tests to confirm this behavior, around here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/test/unit_test/dao_test/participant_dao_test.py#L260

Reply:
Ping.

Reply:
done.